packages/common/core-interfaces/src/logger.ts

```diff
@@ -30,8 +51,15 @@ export interface ITaggedTelemetryPropertyType {
 /**
  * JSON-serializable properties, which will be logged with telemetry.
  */
+export type ITelemetryBaseProperties = ITelemetryProperties;
+
+/**
+ * @see {@link ITelemetryBaseProperties}
+ *
+ * @deprecated Renamed to ITelemetryBaseProperties
```

Comment:
Nit (so the generated API docs contain a nice link to the replacement): 
```suggestion
 * @deprecated Renamed to {@link ITelemetryBaseProperties}
```